internal/format/document.go

```diff
@@ -113,7 +113,7 @@ const (
 					{{- $sensitive := ternary .Sensitive "<sensitive>" .GetValue -}}
 					Value: {{ value $sensitive | sanitizeDoc }}
 
-					Sensitive: {{ ternary (.Sensitive) "yes" "no" }}
+					{{ if $.Settings.ShowSensitivity }}Sensitive: {{ ternary (.Sensitive) "yes" "no" }}{{ end }}
```

Comment:
What do you think about using this instead, it's more readable.

```suggestion
					{{ if $.Settings.ShowSensitivity -}}
						Sensitive: {{ ternary (.Sensitive) "yes" "no" }}
					{{- end }}
```